ci/properties/go-ossf-slsa3-publish.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "SLSA Go releaser",
+    "creator": "Open Source Security Foundation (OpenSSF)",
+    "description": "Compile your Go project using a SLSA3 compliant builder. See https://github.com/slsa-framework/slsa-github-generator to learn more about the buidler, and slsa.dev to learn more about the benefits of SLSA",
+    "iconName": "go-publisher-ossf-slsa3",
```

Comment:
update the icon name here. PR check is failing.

Reply:
done, sorry I forgot to push.